### PR TITLE
chore: add contribution guide and bug reports docs.

### PR DESCRIPTION
**Pull request by raksiv**
Update config.json references.

### CODE REVIEW

File: docs/support/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Steps to reproduce the bug
+- A clear and concise description of what you expected to happen
+- Screenshots (if applicable)
+
+### Include environment and setup information
+
+Any information about your environment and setup will be important to help reproduce the bug. Please include the following:
+
+- Which version of the **Nitric CLI** you are using (run `nitric version` to see your version)
+  - Your **Operating System** and **Go** version
```
Comment (tjholm) [inline docs/support/bug-report.md:42]: We probably don't need their go version, they may not even have it installed.

**Review by tjholm — CHANGES_REQUESTED** [review]
looks great just one comment regarding required info.

File: docs/support/bug-report.md (hunk 1 above)
Comment (raksiv) [reply docs/support/bug-report.md]: Thanks, removed this piece.

**Review by tjholm — APPROVED** [review]

File: doc.config.json [hunk 2]
```diff
@@ -366,11 +366,11 @@
         "support": [
           {
             "title": "Support",
-            "path": "/docs/support/_index.mdx",
+            "path": "/docs/support/support.md",
```
Comment (davemooreuws) [inline doc.config.json:369]: This should still be _index otherwise the url path will be support/support

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: doc.config.json (hunk 2 above)
Comment (tjholm) [reply doc.config.json]: @davemooreuws this should be fixed now

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**